hack/test-cmd.sh

```diff
@@ -148,15 +128,11 @@ openshift start \
   --images="${USE_IMAGES}"
 
 
-# Don't try this at home.  We don't have flags for setting etcd ports in the config, but we want deconflicted ones.  Use sed to replace defaults in a completely unsafe way
-sed -i "s/:4001$/:${ETCD_PORT}/g" ${SERVER_CONFIG_DIR}/master/master-config.yaml
```

Comment:
I think it's the sed -i syntax being unsupported on mac.  You have to do
something else, and there is no one form that works on both.  Perhaps we
need a sed wrapper bash func - os::util::sed <sed args> which adds the
proper -i

On Thu, Sep 24, 2015 at 7:59 AM, David Eads <notifications@github.com>
wrote:

> In hack/test-cmd.sh
> <https://github.com/openshift/origin/pull/4776#discussion_r40310750>:
>
> > @@ -148,15 +128,11 @@ openshift start \
> >    --images="${USE_IMAGES}"
> >
> >
> > -# Don't try this at home.  We don't have flags for setting etcd ports in the config, but we want deconflicted ones.  Use sed to replace defaults in a completely unsafe way
> > -sed -i "s/:4001$/:${ETCD_PORT}/g" ${SERVER_CONFIG_DIR}/master/master-config.yaml
>
> yq comes to mind, but that's not as standard as jq
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4776/files#r40310750>.
>